GiniVision/Classes/Extensions.swift

```diff
@@ -268,4 +268,22 @@ internal extension UIApplication {
     }
 }
 
+internal extension UILabel {
```

Comment:
I thought about it, because I prefer to have a different file for each class extension (all inside _Extensions_ folder). I created the issue so I won't forget about it.